### ISSUE TITLE
File Transport logging colors as 'ESC' and some random strings

### ISSUE THREAD

**Issue by eroongs** [jukkahell/nestlogger#5]
![image](https://user-images.githubusercontent.com/30280255/70625382-40a03980-1c5d-11ea-909a-ca9399944192.png)

Hi! 

Noticed that the log file is also prints the color in the format. 

Just want to ask if there's a way to configure the transports to colorized in the transport console but not in transport file? 

Thanks!

**Comment by jukkahell** [comment]
I'd say it's not possible, but I can dig more into this later if there's a way to do it.

I'm using less -R when viewing logfiles to get colors visible. So if your use case is only to view the logs (without those ugly escapes) there could be a way for your viewer to get the colors handled properly.

**Comment by jukkahell** [comment]
Got some time to look into this and it is actually possible to do. You still need this feature?

**Comment by eroongs** [comment]
Hi!

Yes. I think it would be better if you could have this feature as configurable.

Also, Is it possible to have multiple log files using this plugin/library? If yes, how? 

Cos Loggers in the LoggerService only supports CONSOLE and only 1 ROTATE LOG. 

I wish to create multiple log files for separation of concerns e.g ( query.log, api.log, http.log )

**Comment by jukkahell** [comment]
Will do then. I think multiple log files is possible as well. I have to change a bit the constructor to support more complex logger configurations. I'll let you know when these are done.

**Comment by jukkahell** [comment]
Published version 4.0.1 with (hopefully) better way to configure loggers. Updated the readme as well but here's shortly how you'd initialize the logger in your module in the newest version:
```javascript
   const loggers = LoggerService.getLoggers(
      config.logAppenders,
      { serviceName: config.serviceName, path: config.logFilePath, colorize: config.colorize }
   );
   return new LoggerService(
      config.logLevel,
      loggers,
   );
```
You can also do it like this if you want different options for console and file:
```javascript
   const loggers = [
      LoggerService.console({ serviceName: config.serviceName, timeFormat: "HH:mm" }),
      LoggerService.rotate({ serviceName: config.serviceName, path: config.logFilePath, colorize: false }),
   ];
   return new LoggerService(
      config.logLevel,
      loggers,
   );
```
And with regard to how you can have multiple log files. You should create one module for each file you want. Just give different serviceName for each rotate appender. Then you import the specific logger module in the module where it is needed.
